docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -37,6 +37,12 @@ They are implemented as methods in the [test controller](../../reference/test-ap
 
 The available actions with reproducible usage examples and links to their descriptions are listed below.
 
+> TestCafe is intended to emulate real user behavior, so you cannot interact with elements, that are not visible at that moment.
```

Comment:
I feel that we need to add a link to the article that describes visibility.

Reply:
Done that